src/components/user/DeleteAccountPage.js

```diff
@@ -0,0 +1,168 @@
+    minHeight: '100vh', // Set minimum height to cover the entire viewport
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    overflow: 'hidden',
+});
+
+// Main Content Area Style
+const mainContentStyle = () => ({
+    backgroundColor: 'rgba(91, 46, 72, 1)',
+    width: '100%',
+    minHeight: '60vh',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-around',
+    alignItems: 'center',
+    position: 'relative',
+    overflow: 'hidden',
+    padding: '20px',
+    boxSizing: 'border-box',
+});
+
+// Group of Texts Style
+const textGroupStyle = () => ({
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: '100%',
+});
+
+// Logo Image Style
+const logoImageStyle = () => ({
+    marginTop: '10px',
+    width: '180px',
+    height: 'auto',
+    marginBottom: '50px',
+});
+
+// Trash Icon Style
+const trashIconStyle = () => ({
+    marginTop: '30px',
+    filter: 'invert(100%) sepia(0%) saturate(0%) hue-rotate(0deg) brightness(100%) contrast(100%)',
+});
+
+// Header Text Style
+const headerTextStyle = () => ({
+    color: 'rgba(255, 255, 255, 1)',
+    textAlign: 'center',
+    fontFamily: 'Montserrat',
+    fontSize: '65px',
+    marginTop: '10px',
+});
+
+// Informative Text Style
+const infoTextStyle = () => ({
+    color: 'rgba(255, 255, 255, 1)',
+    textAlign: 'center',
+    fontFamily: 'Inknut Antiqua',
+    fontSize: '40px',
+    marginTop: '40px',
+});
+
+// Confirmation Input Style
+const confirmationInputStyle = () => ({
+    padding: '10px',
+    borderRadius: '20px',
+    marginTop: '30px',
+    width: '500px',
+    boxShadow: '0px 4px 8px rgba(0, 0, 0, 0.8)',
+});
+
+// Confirmation Text Style
+const confirmationTextStyle = () => ({
+    textAlign: 'center',
+    color: 'rgba(255, 255, 255, 1)',
+    fontFamily: 'Inknut Antiqua',
+    fontSize: '24px',
+    fontWeight: 700,
+    marginTop: '80px',
+});
+
+// Button Container Style
+const buttonContainerStyle = () => ({
+    display: 'flex',
+    justifyContent: 'space-between',
+    width: '600px',
+    marginTop: '30px',
+    padding: '0 20px',
+});
+
+// Action Button Style
+const actionButtonStyle = () => ({
+    backgroundColor: 'rgba(91, 46, 72, 1)',
+    width: '250px',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: '10px 45px',
+    borderRadius: '20px',
+    marginBottom: '10px',
+    marginTop: '40px',
+    boxShadow: '0px 4px 8px rgba(0, 0, 0, 1)',
+});
+
+// Button Text Style
+const buttonTextStyle = () => ({
+    color: 'white',
+    fontSize: '16px',
+    fontWeight: 'bold',
+});
+
+// Main Account Deletion Page Component
+const AccountDeletionPage = () => {
+  const navigate = useNavigate(); 
+
+  // Navigate back to home after deletion
+  const handleDelete = () => {
+      console.log("Account is being deleted...");
+      // Add deletion logic here
```

Comment:
we have to ensure that logic is implemented fully before a pull request to the main branch is done